src/command_modules/azure-cli-datalake/azure/cli/command_modules/datalake/analytics/_help.py

```diff
@@ -0,0 +1,241 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.help_files import helps #pylint: disable=unused-import
+
+#pylint: disable=line-too-long
+# TODO: Understand the pattern for help. Is it allowed to have ADLA and ADLS in the same "command"?
+helps['datalake'] = """
+    type: group
+    short-summary: Access to Data Lake Store and Analytics management
+    long-summary: If you don't have the datalake component installed, add it with `az component update --add datalake`
+"""
+
+helps['datalake analytics'] = """
+    type: group
+    short-summary: Commands to manage Data Lake Analytics accounts, jobs and catalogs. 
+"""
+
+helps['datalake analytics job'] = """
+    type: group
+    short-summary: Commands to manage Data Lake Analytics jobs. 
+"""
+
+helps['datalake analytics job submit'] = """
+    type: command
+    short-summary: submits the specified job to the specified Data Lake analytics account. 
+    parameters:
+        - name: --job-name
+          type: string
+          short-summary: 'Job name for the job'
+        - name: --script
+          type: string
+          short-summary: 'The script to submit'
+          long-summary: This can be either the script contents or a valid file path to a file containing the script
+        - name: --runtime-version
+          short-summary: 'The runtime version to use'
+          long-summary: This parameter is used for explicitly overwriting the default runtime. It should only be done if you know what you are doing.
+        - name: --degree-of-parallelism
+          short-summary: 'The degree of parallelism for the job'
+          long-summary: Higher values equate to more parallelism and will usually yield faster running jobs, at the cost of more AUs consumed by the job.
+        - name: --priority
+          short-summary: 'The priority of the job'
+          long-summary: Lower values increase the priority, with the lowest value being 1. This determines the order jobs are run in.
+
+"""
+
+helps['datalake analytics job cancel'] = """
+    type: command
+    short-summary: cancels the specified job in the specified Data Lake analytics account. 
```

Comment:
Haha fair point, This was a help copy/paste from PowerShell, so that our help looks the same. I can remove it though and just say "cancels a job in the specified Data Lake..."